vital/types/detected_object_set.cxx

```diff
@@ -209,6 +222,42 @@ select( const std::string& class_name, double threshold )const
   return vect;
 }
 
+// ------------------------------------------------------------------
+void
+detected_object_set::
+scale( double scale_factor )
+{
+  if( scale_factor == 1.0 )
+  {
+    return;
+  }
+
+  VITAL_FOREACH( auto detection, m_detected_objects )
+  {
+    auto bbox = detection->bounding_box();
```

Comment:
It might be useful to add an operator on a detected_object to scale the bbox. - e.g.

    void scale( detected_object& detection, double scale_factor )
    {
        auto bbox = detection.bounding_box();
        bbox = kwiver::vital::scale( bbox, scale_factor );
        detection.set_bounding_box( bbox );
    }

The same could be done for shift().

Similarly, these methods could be converted into operators so they are not part of the object.